crates/open_ai/src/open_ai.rs

```diff
@@ -113,6 +117,15 @@ impl Model {
             Self::Custom { max_tokens, .. } => *max_tokens,
         }
     }
+
+    pub fn max_output_tokens(&self) -> Option<u32> {
+        match self {
+            Self::Custom {
+                max_output_tokens, ..
+            } => *max_output_tokens,
+            _ => None,
```

Comment:
any chance you can include the max output tokens for the other models here? they can be found in the lists on this page: https://platform.openai.com/docs/models

Reply:
Thank you for the suggestion! I have added support for the max_output_tokens for the relevant models as per the documentation. However, I don’t have the OpenAI tokens to conduct testing myself. If someone could help with the testing, I would greatly appreciate it!